src/helpers/eventBroadcasterMain.js

```
@@ -11,6 +11,10 @@ const getEndpoints = (eventGroup) => {
 		case 'userEvents':
 			if (process.env.EVENT_USER_LISTENER_API)
 				return process.env.EVENT_USER_LISTENER_API.split(',').filter((url) => url.trim())
+		case 'tenantEvents':
+			if (process.env.EVENT_TENANT_LISTENER_API)
+				return process.env.EVENT_TENANT_LISTENER_API.split(',').filter((url) => url.trim())
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)